### PR TITLE
docs: add Installation notes to README

### PR DESCRIPTION
**Pull request by rpappalax**
@bbangert r?

### CODE REVIEW

**Comment by rpappalax** [comment]
looks like travis wasn't able to wget the pypy bz2 file.  can you retry the travis build (only repo owners can do this) or reject the pr and i'll resubmit?

File: setup.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
         "service-identity>=14.0.0",
         "treq>=15.0.0",
         "pyOpenSSL>=0.15.1",
-        "txaio==2.1.0",
+        "txaio==2.2.0",
```
Comment (bbangert) [inline setup.py]: Why the bump here? I froze it at 2.1.0 because on linux it threw an error using 2.2.0, I'll re-test.

Comment (rpappalax) [inline setup.py]: I couldn't get it to work on OSX with 2.1.0 hence the bump.  I can try both again today if that's the case

Comment (bbangert) [inline setup.py]: Ah, interesting. On linux I was getting an error about txaio not having a method called when running it.

**Pull request merged**